fastapi_sse.py

```diff
@@ -84,10 +85,12 @@ def sse_response(
 
     async def event_source_wrapper():
         async for event in generator:
+            data = event.model_dump_json() if not isinstance(event, Sequence) else json.dumps([e.model_dump() for e in event])
```

Comment:
What about using `TypeAdapter(list[dict]).dump_json([e.model_dump() for e in events])` via `from pydantic import TypeAdapter`?

This allows us to still use pydantic's json serializer which is theoretically faster and also makes the formatting consistent (so you'll need to change the test cases to have no spaces in the json).